server/paymentProviders/paypal/payouts.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable @typescript-eslint/camelcase, camelcase */
+
+import { isNil, round, toNumber } from 'lodash';
+import moment from 'moment';
+
+import activities from '../../constants/activities';
+import status from '../../constants/expense_status';
+import * as paypal from '../../lib/paypal';
+import { createFromPaidExpense as createTransactionFromPaidExpense } from '../../lib/transactions';
+
+export const payExpensesBatch = async (expenses: any[]): Promise<any[]> => {
+  const [firstExpense] = expenses;
+  const isSameHost = expenses.every(e => !isNil(e.CollectiveId) && e.CollectiveId === firstExpense.CollectiveId);
+  if (!isSameHost) {
+    throw new Error('All expenses in the batch should belong to the same Collective.');
+  }
+
+  const host = await firstExpense.collective.getHostCollective();
+  if (!host) {
+    throw new Error(`Could not find the host embursing the expense.`);
+  }
+
+  const [connectedAccount] = await host.getConnectedAccounts({
+    where: { service: 'paypal', deletedAt: null },
```

Comment:
Let's be more specific, we also have some entries for `service=paypal type=payment` in the DB used when people pay with PayPal.

```suggestion
    where: { service: 'paypal', type: 'adaptive', deletedAt: null },
```

Reply:
@Betree correct me if I'm wrong, but aren't those saved as Payment Methods?

Reply:
Yes, my bad! :slightly_smiling_face: